### ISSUE TITLE
[Detailslist_columnheader] Cannot tab to the info icon in column header to make the tooltip pop up 

### ISSUE THREAD

**Issue by Luciasea** [microsoft/fluentui#21132]
### Environment Information

- **Package version(s)**: any version
- **Browser and OS versions**:  Edge, Chrome

### Please provide a reproduction of the issue in a codepen: https://codepen.io/Luciasea/pen/jOGYgOB
![image](https://user-images.githubusercontent.com/96798634/147613673-8040cc49-f6a8-40b2-b1b8-61940ac953ad.png)

**Comment by gouttierre** [comment]
@Luciasea - Thanks for filing this issue with us. To set expectations, the developers will review this issue once capacity allows as we’re currently focused on larger coherence work items.

@TristanWatanabe - Would you be able to confirm if this is a regression, or if this behavior is an issue? 
cc @smhigley

**Comment by TristanWatanabe** [comment]
@Luciasea  I can confirm that this is indeed occurring. However, `DetailsList` column header names was designed to only take `string` variables; so there's no logic currently in place to properly handle the case where you pass in a fragment with a name containing an icon wrapped in a `TooltipHost`. If this was ever implemented, ideally, the column header would have an `aria-describedby` attribute pointing it to the `id` of the `Tooltiphost` so the contents of the tooltip is read by screenreaders.
